data/json/proficiencies/melee_weapons.json

```
@@ -0,0 +1,778 @@
+[
+  {
+    "type": "proficiency",
+    "id": "prof_auto_rifles_familiar",
+    "//": "Maybe scrap this.",
+    "category": "prof_combat",
+    "name": { "str": "Automatic Rifle Familiarity" },
+    "description": "You're familiar with how to hold an automatic rifle, use a bayonet, and not cut yourself.  In case you need to club someone with it instead of just shooting them, I guess.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_auto_rifles_pro",
+    "//": "Maybe scrap this.",
+    "category": "prof_combat",
+    "name": { "str": "Automatic Rifle Proficiency" },
+    "description": "You're proficient with the usage of an automatic rifle, and feel comfortable with one in your hand.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "16 h",
+    "required_proficiencies": [ "prof_auto_rifles_familiar" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_auto_rifles_master",
+    "//": "Maybe scrap this.",
+    "category": "prof_combat",
+    "name": { "str": "Automatic Rifle Mastery" },
+    "description": "You've mastered the use of all kinds of automatic rifles for the purposes of killing someone without firing it.",
```

```suggestion
    "description": "You've mastered the use of all kinds of automatic rifles for various combat situations.",
```
This proficiency is for firing the gun, correct? Why would the description describe it as a melee proficiency for rifle use?

---

It is for use as a melee weapon, Krav Maga supports using auto-loading rifles/pistols as melee weapons.